packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -311,6 +323,49 @@ class Localizations extends StatefulWidget {
     assert(delegates.any((LocalizationsDelegate<dynamic> delegate) => delegate is LocalizationsDelegate<WidgetsLocalizations>));
   }
 
+  /// Overrides the inherited [Locale] or [LocalizationsDelegate]s for `child`.
+  ///
+  /// This factory constructor is used for the - usually rare - situtation where part
+  /// of an app should be localized for a different locale than the one defined
+  /// for the device, or a different list of [LocalizationsDelegate]s than the
+  /// list defined by [WidgetsApp.localizationsDelegates].
+  ///
+  /// For example you could specify that `myWidget` was only to be localized for
+  /// the US English locale:
+  /// ```
```

Comment:
blank line before back ticks, plus use triple-backtick followed by the word "dart" as above